scripts/detection/faster_rcnn/train_faster_rcnn.py

```diff
@@ -19,6 +20,7 @@
     FasterRCNNDefaultValTransform
 from gluoncv.utils.metrics.voc_detection import VOC07MApMetric
 from gluoncv.utils.metrics.coco_detection import COCODetectionMetric
+import horovod.mxnet as hvd
```

Comment:
Could u check if horovod is installed first? If it is not installed on user's system, it will prevent the user from using the script.

Reply:
Done